doc/data/messages/m/method-hidden/good.py

```diff
@@ -1 +1,7 @@
-# This is a placeholder for correct code for this message.
+class Fruit:
+    def __init__(self, worm):
+        if worm:
+            self.worm = worm
+
+    def bore(self):
+        pass
```

Comment:
```suggestion
    def __init__(self, vitamins):
        self.vitamins = vitamins

    def antioxidants(self):
        pass
```

Reply:
To give the reasoning on that one, ``bore`` should not be a function of the fruit, the insect is boring not the apple.